### PR TITLE
add user and school model

### PR DESCRIPTION
**Pull request by remi-boivin**

### CODE REVIEW

File: models/School.js [hunk 1]
```diff
@@ -5,12 +5,15 @@ const SchoolSchema = new mongoose.Schema({
...
   },
-  Address: {
+  address: {
     type: String,
   },
   phoneNumber: {
     type: Number,
   },
+  riderId: {
+    type: Array,
```
Comment (jasonLavigneDev) [inline models/School.js]: type : ObjectId

File: models/User.js [hunk 2]
```diff
@@ -4,18 +4,31 @@ import mongoose from 'mongoose'
...
+    required: [true, 'Please provide a email.'],
+    maxlenght: [20, 'Email cannot be more than 20 characters'],
   },
   password: {
     type: String,
+    required: [true, 'Please provide a password.'],
   },
-  email: {
+  phoneNumber: {
     type: String,
```
Comment (jasonLavigneDev) [inline models/User.js]: type : Number

File: models/User.js [hunk 3]
```diff
@@ -4,18 +4,31 @@ import mongoose from 'mongoose'
...
+    required: [true, 'Please provide a phone number.'],
+    maxlenght: [20, 'Phone number cannot be more than 20 characters'],
   },
-  school: {
-    type: Array,
+  schoolId: {
+    type: String,
```
Comment (jasonLavigneDev) [inline models/User.js]: type : Object Id

**Review by jasonLavigneDev — CHANGES_REQUESTED** [review]
Attention pour le type ObjectId de mongoose il faut parfois importer l objet 

import mongoose, { ObjectId } from "mongoose";

File: models/School.js (hunk 1 above)
Comment (remi-boivin) [reply models/School.js]: it's an array. We need to have all rides for a school :)

**Pull request merged**